crates/polars-ops/src/series/ops/search_sorted.rs

```diff
@@ -12,7 +12,7 @@ pub fn search_sorted(
     let original_dtype = s.dtype();
 
     if s.dtype().is_categorical() {
-        // See https://github.com/pola-rs/polars/issues/20171
+        // Hopefully to be fixed in https://github.com/pola-rs/polars/pull/22633
```

Comment:
Can you revert this comment? Let's stick to referring to issues rather than potential solutions.